### PR TITLE
Update to ecto 2.1.0-rc.3

### PR DESCRIPTION
**Pull request by wojtekmach**
Just testing the latest rc ;-)

### CODE REVIEW

File: priv/repo/migrations/20161024204753_remove_package_downloads_pk.exs [hunk 1]
```diff
@@ -0,0 +1,84 @@
+defmodule HexWeb.Repo.Migrations.RemovePackageDownloadsPk do
+  use Ecto.Migration
+
+  def up do
```
Comment (wojtekmach) [inline priv/repo/migrations/20161024204753_remove_package_downloads_pk.exs]: @ericmj could you double-check this change especially? I basically combined these 2 migrations:
- https://github.com/hexpm/hex_web/blob/master/priv/repo/migrations/20160317073758_add_package_downloads_pk.exs
- https://github.com/hexpm/hex_web/blob/master/priv/repo/migrations/20160530111051_optimize_downloads_view_indicies.exs

Comment (ericmj) [inline priv/repo/migrations/20161024204753_remove_package_downloads_pk.exs]: Maybe it's not worth it just to drop the primary key from the view or is there some specific reason?

Comment (wojtekmach) [inline priv/repo/migrations/20161024204753_remove_package_downloads_pk.exs]: no reason, we simply don't need the pk anymore but it doesn't hurt to keep it around either

Comment (ericmj) [inline priv/repo/migrations/20161024204753_remove_package_downloads_pk.exs]: Okay, lets hold off for now but add a note to the schema that we can remove it.

**Pull request merged**